index.js

```diff
@@ -11,14 +12,40 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+const maxPage = 42;
```

Comment:
Please do not hardcore `maxPage` - this value you can get from the API and might change.